src/main.js

```diff
@@ -9,3 +8,41 @@ import 'o-layout';
 import './js/demo';
 import './js/main';
 import 'o-autoinit';
+import oTracking from 'o-tracking';
+import CookieMessage from 'o-cookie-message';
+
+const trackingCookieName = 'OrigamiCookieConsentGDPR';
+
+new CookieMessage(null, {
+	consentCookieName: trackingCookieName
```

Comment:
Hmm, is it possible to regect tracking for this cookie name? I think this cookie message will show the default content with a "manage cookies" button which will go to an FT cookie consent page. Since the policy mentions "Operating our Sites" and this is an ft.com domain maybe we can leave the default cookie name?